Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/PersonOverviewViewModel.kt

```diff
@@ -20,22 +27,43 @@ import de.rki.coronawarnapp.util.coroutine.AppScope
 import de.rki.coronawarnapp.util.coroutine.DispatcherProvider
 import de.rki.coronawarnapp.util.ui.SingleLiveEvent
 import de.rki.coronawarnapp.util.viewmodel.CWAViewModel
-import de.rki.coronawarnapp.util.viewmodel.SimpleCWAViewModelFactory
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactory
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.flow.combine
 import kotlinx.coroutines.flow.onStart
 import timber.log.Timber
 
+@Suppress("LongParameterList")
```

Comment:
Hmm, this ViewModel has a lot of depdencies now... Do you think it makes sense to create an additional component like `AddmissionTileProvider` that contains this new logic?

Reply:
makes sense?

Reply:
done, but the annotation stays anyway 🤷‍♂️

Reply:
Thanks